apps/frontend/src/app/layout.tsx

```diff
@@ -28,7 +28,7 @@ export default function RootLayout({
       <body
         className={`${inter.className} min-h-dvh flex flex-col justify-between`}
         style={{
-          background: "url('bg.png')",
+          background: "url('/bg.png')",
```

Comment:
I hope this doesn't break anything, but for me there was no background on subpages, because it was trying to download /periods/1/bg.png. Doesn't happen in prod, but this should work everywhere

Reply:
am I tripping or are the backgrounds slightly different?

https://schbody-frontend-git-feature-pass-export-optimize-kir-dev.vercel.app/

https://body.sch.bme.hu/